.github/workflows/ci.yml

```
@@ -32,17 +60,50 @@ jobs:
           # Make sure the database is actually working
           psql "${CRATESFYI_DATABASE_URL}"
 
-      - name: Build
-        run: cargo build --workspace --locked
+      - name: run tests
+        shell: bash
+        run: |
+          for f in ./test-binaries/*; do
+            echo "running $f"
+            $f || break
```

I don't think `break` will fail the test.
```suggestion
            $f || exit 1
```